arch/arm/core/swi_tables.ld

```diff
@@ -3,6 +3,6 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-#if LINKER_ZEPHYR_FINAL
+#if LINKER_ZEPHYR_FINAL && CONFIG_ISR_TABLES_LOCAL_DECLARATION
```

Comment:
Does this work if `CONFIG_ISR_TABLES_LOCAL_DECLARATION` isn't defined?

```suggestion
#if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
```

Reply:
C preprocessor treads undefined symbols as 0 in such situation. defined(...) is here to be able to use undefined preprocessor macros in C conditionals. I can change it anyway.

Reply:
Done